x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/common.ts

```diff
@@ -130,6 +130,6 @@ export function isCompletedAnalyticsJob(stats: DataFrameAnalyticsStats) {
   return stats.state === DATA_FRAME_TASK_STATE.STOPPED && progress === 100;
 }
 
-export function getResultsUrl(jobId: string, analysisType: string) {
+export function getResultsUrl(jobId: string, analysisType: ANALYSIS_CONFIG_TYPE | string) {
```

Comment:
nit, and probably outside the scope of this PR, but it would be good if this only took `ANALYSIS_CONFIG_TYPE` 
it looks like `getAnalysisType` should return a `ANALYSIS_CONFIG_TYPE` rather than a string.

Reply:
Yeah I noticed that, too, but it will involve touching more files so can be done in a separate PR.